<launch>
    <arg name="namespace" default=""/>
    <arg name="proj_dir" default="$(find vibro_tactile_toolbox)"/>
    <group ns="$(arg namespace)">
        <node pkg="vibro_tactile_toolbox" name="save_hande_pose" type="save_hande_pose.py" output="screen">
            <param name="namespace" value="$(arg namespace)"/>
            <param name="root_pwd" value="$(arg proj_dir)"/>
        </node>
    </group>
</launch>
